### PR TITLE
updates to horizon ops

### PR DESCRIPTION
**Pull request by oryband**
also add new apt-get-upgrade role

### CODE REVIEW

File: deploy/ansible/playbooks/group_vars/example/horizon.yml [hunk 1]
```diff
@@ -20,3 +20,4 @@ horizon:
           user: stellar
           password: xxx
           host: xxx
+          cursor_name: 'HORIZON'
```
Comment (fuzzyami) [inline deploy/ansible/playbooks/group_vars/example/horizon.yml]: @oryband I'd rather this was somehow unique or at least document that it needs to be unique.

File: deploy/ansible/playbooks/group_vars/example/horizon.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ horizon:
   nodes:
     - name: local
       aws_region: us-east-1
-      s3_bucket_url: s3://something #NOTE: you'll need to manually allow the node to access this bucket. terraform doesn't do thisyet.
```
Comment (fuzzyami) [inline deploy/ansible/playbooks/group_vars/example/horizon.yml]: @oryband terraform doesnt (yet) allow writing from the *horizon* into the bucket. only from the core. this comment should remain. horizon doesn't normally need to write into this bucket - only when its dumping logs/its db (initiated by a script).

Comment (oryband) [reply deploy/ansible/playbooks/group_vars/example/horizon.yml]: if horizon needs to write to any bucket, it should be its own.
- two apps shouldn't interfere with each other's storage.
- what's more, that bucket is reserved for history and can be copied by other nodes that want to cathcup form this one. i wouldn't want to store logs (with potentiall sensitive things going on) to a public bucket.

File: deploy/ansible/playbooks/group_vars/example/horizon.yml (hunk 1 above)
Comment (oryband) [reply deploy/ansible/playbooks/group_vars/example/horizon.yml]: done

**Pull request merged**